api_io_sign.yaml

```diff
@@ -67,12 +67,86 @@ paths:
           schema:
             $ref: "#/definitions/ProblemJson"
 
+  "/signatures":
+    post:
+      operationId: createSignature
+      summary: Create a Signature from SignatureRequest
+      parameters:
+        - in: body
+          name: signatureToCreate
+          required: true
+          schema:
+            $ref: '#/definitions/CreateSignatureBody'
+      responses:
+        "200":
+          description: Signature created
+          schema:
+            $ref: "#/definitions/SignatureDetailView"
+        "400":
+          description: Validation error.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "401":
+          description: Bearer token null or expired.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "403":
+          description: Forbidden.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "404":
+          description: Resource not found.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "502":
+          description: Bad gateway.
+        "504":
+          description: Gateway timeout.
+        "500":
+          description: Service unavailable.
+          schema:
+            $ref: "#/definitions/ProblemJson"
 
+  "/signature-requests/{id}":
+    post:
+      operationId: getSignatureRequestById
+      summary: Get a Signature Request from id
+      parameters:
+        - in: path
+          name: id
+          type: string
```

Comment:
can we use a stricter definition in order to prevent path traversal?

Reply:
minLength and maxLength could be enough?

Reply:
I've double checked this, since io-sign API do not use the in-path user identifier to authenticate him, it's not an issue.
I'd add `minLength` anyway since it looks like it's used here: https://github.com/pagopa/io-sign/blob/main/apps/io-func-sign-user/openapi.yaml#L286